### ISSUE TITLE
Clicking on the UO shield causes the menu to appear only partly/with wrong size.

### ISSUE THREAD

**Issue by transentia** [nikrolls/uBlock-Edge#109]
Read first: <https://github.com/nikrolls/uBlock-Edge/blob/master/CONTRIBUTING.md>

### Describe the issue
Clicking on the UO shield causes the menu to appear only partly/with wrong size.

### One or more specific URLs where the issue occurs

[URL(s) for issue on a specific site are **mandatory**]
Can be seen on THIS page: https://github.com/nikrolls/uBlock-Edge/issues/new

### Screenshot in which the issue can be seen

[Screenshot(s) for visual issues are **mandatory**]
![partial](https://user-images.githubusercontent.com/105110/37649414-74cd6f38-2c7d-11e8-82cd-4258fea634e9.png)

![bigger](https://user-images.githubusercontent.com/105110/37649775-9e87ad88-2c7e-11e8-93b0-8d9cba213617.PNG)

### Steps for anyone to reproduce the issue
Click on the UO shield
Menu appears; menu width is usually only about 1/2 the 'normal' width of the left-hand column (see 'partial') 
The blue 'power' icon flashes before the screen is redrawn with the contents of the left-hand grey grid.
Sometimes, when the shield has been clicked a number of times (trying to get it to show up correctly), the menu appears oversized (see 'bigger').
Doesn't make any difference if Edge window is maximised or not, or if the windows is positioned at very top-right of screen or away from the edges of the screen.
Doesn't make any difference if a tab is torn off into a new window.
Doesn't make any difference if window is brand new or if UO has blocked content
The "show next to address bar" setting makes no difference.

### Your settings

[If you fail to provide this info, the issue will be marked as invalid. Lists all settings which differ from default]

Default settings...new install.

- OS/version: win 10, all latest patches. Dell XPS 15, 9550
- Browser/version: Microsoft Edge 41.16299.248.0, Microsoft EdgeHTML 16.16299
- uBlock Origin version: 1.15.4

**Comment by transentia** [comment]
Forgot to say that the XPS 15 is a hiDPI display. that's probably the reason for this.

**Comment by IcePanther** [comment]
Yeah, I have the same problem since a few days, maybe it's related to a Windows update ?

Anyway, it does this OR still opens too small as it did before, the previous fix (for issues 70, 95, etc.) didn't work on my machines (Surface Pro 3 - HiDPI ; tower, not HiDPI)

**Comment by nikrolls** [comment]
There hasn't been a release since #70 and #95 were merged, so they won't have shown up yet.

More accurately, there were a few uBlock releases but they dealt with Firefox only and I didn't do an Edge release for them. I've submitted the latest stable uBlock release to the Store now so once that's publicly available I'll loop back to see if the issue is resolved for you all.

**Comment by nikrolls** [comment]
@transentia @IcePanther, can you please try with the latest update available now, and confirm if this has been fixed?

**Comment by transentia** [comment]
This looks better, but still isn't 100%.

The UI mostly shows without the left-hand grid but once every 2 or 3 clicks, the UI shows the grid. In my testing it always shows the 'power' column. It has shown up too-large once or twice as well...but right-justified in the space, rather than left-justified as in my earlier screenshot.

Also: when I click, I can see a very brief redraw in the too-large region of the screen before the actual UI shows up.

It IS improved but I'm guessing that HiDPI is still a problem... 

Thanks.

**Comment by IcePanther** [comment]
@nikrolls 

For me, on the two items :

- Regarding the #70, #95 etc. problem : Now it's kind of fixed, but not completely : When the popup opens, it's still "too small" (the grid doesn't show) randomly (more than half the time) but at least now it's properly aligned so the base controls show up, and I have to click twice on "blocked requests" to make the grid appear.
The base UI always appear in full and when the window doesn't properly shows the grid, it still show it correctly (so the basic UI is displayed OK 100% of the time). It's only the grid that does not always show up.
While this is not a complete fix, it's still way better than the previous situation, where the controls were completely unusable, and one had to click again and again until the controls would appear correctly.
Now it's not perfect but definitely usable, thank you for publishing the fix :)

- Regarding the HiDPI problem, I cannot reproduce it anymore on the Surface Pro. it acts the same as the not-HiDPI screen, which is, as described in the above paragraph.
